core/app/models/spree/address.rb

```diff
@@ -9,7 +9,7 @@ class Address < Spree::Base
     has_many :cartons, inverse_of: :address
     has_many :credit_cards, inverse_of: :address
 
-    validates :firstname, :lastname, :address1, :city, :country, presence: true
+    validates :firstname, :lastname, :address1, :city, :country_id, presence: true
```

Comment:
Don't use `validates_associated`, we don't need to validate the `country` model to create the address model. `Country` records rarely change, and aren't created at the same time as Addresses